package-lock.json

```diff
@@ -0,0 +1,2433 @@
+{
```

Comment:
I think this file should go away. What do you think? from what I understand, if you do `npm install i` it should install all the packages and their dependencies and create a reference in this lock file, right?

Reply:
Yes

Reply:
Okay, that makes sense ` so its work is to ensure that as it works in my system so it will work in yours` but we should all use same node version or better you can specify the node engine like so:

```json
  "engines": {
    "npm": ">=5.5",
    "node": ">=6.12"
  },
```

in package.json file